### PR TITLE
Added attack and html, css

### PR DESCRIPTION
**Pull request by zhangji52**

### CODE REVIEW

**Comment by zhangji52** [comment]
I implemented some attack functions and HTML and CSS. I changed the square columns (game interface) to the middle of the screen, added animation when hovering the battleship buttons. Also added a background image for the players to enjoy a better gaming experience. Changed some HTML for the whole program format.

**Review by guyera — CHANGES_REQUESTED** [review]
EDIT: Ignore what I said about doing a git pull. I went ahead and merged it, and ran a diff, and as suspected, your Board.java and Result.java are the exact same as what was already on the remote master branch. I'm not sure why they were showing up as changed in the PR diff files. Perhaps in the future, be sure to do a git pull to make sure all your files are up to date before doing a push.

This looks great! However, the attack function is already implemented, and the Result class is already done. Those changes you made have already been made. Make sure your local repository is up to date with the remote repository.

Basically, I need you to do a git pull to update your Board.java and Result.java classes to be consistent with the ones on the remote repo, as they are already done and functioning perfectly. I do not want to merge any more changes of those files into the master branch. The HTML / CSS looks great though, so once you do a git pull and make your Board.java and Result.java files consistent with the remote master branch's ones, then make another pull request.

**Review by guyera — COMMENTED** [review]
I'll go ahead and merge this, and then I'll run a local check to make sure the Board.java and Result.java files are the same as what's already in the master branch. If not, I'll re-merge the correct files.
